types/account.d.ts

```diff
@@ -70,6 +70,11 @@ export type RegisterPayload = PayloadWithIPOverride & {
   externalData?: string;
 };
 
+export type RegisterArgs = {
+  config: Config;
+  user: RegisterPayload;
+};
+
 export type CaptureFirstNameLastName = {
```

Comment:
Let's move all these payload types into the cleeng.d.ts file and avoid manipulating them.

Reply:
Moving these types in the cleeng.d.ts somehow interferes the request types that are defined there and requires changing through the entire code.

Reply:
Will leave this for later stage.